online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZView.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 8/9/2022.
+ */
+
+package com.adyen.checkout.onlinebankingcz
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.AdapterView
+import androidx.lifecycle.LifecycleOwner
+import com.adyen.checkout.components.PaymentComponentState
+import com.adyen.checkout.components.model.payments.request.OnlineBankingCZPaymentMethod
+import com.adyen.checkout.components.ui.adapter.SimpleTextListAdapter
+import com.adyen.checkout.components.ui.view.AdyenLinearLayout
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import com.adyen.checkout.onlinebankingcz.databinding.OnlineBankingCzSpinnerLayoutBinding
+
+class OnlineBankingCZView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) :
+    AdyenLinearLayout<
+        OnlineBankingOutputData,
+        OnlineBankingConfiguration,
+        PaymentComponentState<OnlineBankingCZPaymentMethod>,
+        OnlineBankingCZComponent
+        >(context, attrs, defStyleAttr),
+    AdapterView.OnItemSelectedListener {
+
+    private val binding: OnlineBankingCzSpinnerLayoutBinding =
+        OnlineBankingCzSpinnerLayoutBinding.inflate(LayoutInflater.from(context), this)
+
+    private val issuersAdapter: SimpleTextListAdapter<OnlineBankingModel> = SimpleTextListAdapter(context)
```

Comment:
This adapter means that the icons of the banks won't be appearing right? Should we make a custom adapter that inflates custom views and shows the icon as well, as per the design?
![Screenshot_1663078837](https://user-images.githubusercontent.com/9079915/189926464-15218f14-0da9-4919-9c98-fcd837b123d4.png)

Reply:
Oops I was looking at PL 🤦